packages/arb-token-bridge-ui/.e2e.env.sample

```diff
@@ -1,2 +1,4 @@
 NETWORK_NAME=goerli
 SECRET_WORDS="test, test, test, test, test, test, test, test, test, test, test, junk"
+ADDRESS="0xyour_address_here"
+INFURA_KEY=process.env.REACT_APP_INFURA_KEY
```

Comment:
We can remove this completely and just use `REACT_APP_INFURA_KEY`.

Reply:
Yup we can remove this line
fyi we can't use `process.env` directly
https://stackoverflow.com/questions/57818181/how-to-use-process-env-variables-in-browser-running-by-cypress